src/components/picker/types.ts

```diff
@@ -237,12 +252,20 @@ export type PickerBaseProps = Omit<TextFieldProps, 'value' | 'onChange'> &
   };
 
 export type PickerPropsWithSingle = PickerBaseProps & {
+  /**
+   * Custom top element
+   */
+  customTopElement?: () => React.ReactElement;
```

Comment:
You're calling the prop `customTopElement` but it's actually a `renderCustomTopElement` (it's a function)